vmcheck/vmcheck_linux_amd64.go

```diff
@@ -48,12 +48,12 @@ func hypervisorPortCheck() (bool, error) {
 	}
 
 	p := &bdoor.BackdoorProto{}
-	p.CX.Low.SetWord(bdoor.CommandGetVersion)
 
+	p.CX.Low.SetWord(bdoor.CommandGetVersion)
 	out := p.InOut()
+	return (0xffffffff != out.AX.Low.Word()) && (0 != out.AX.Low.Word()), nil
 
 	// if there is no device, we get back all 1s
```

Comment:
Small nit.  This comment is actually pretty useful and should probably go above the line at 54.